egui/src/containers/collapsing_header.rs

```diff
@@ -89,6 +89,14 @@ impl CollapsingState {
 
     /// Will toggle when clicked, etc.
     fn show_default_button_indented(&mut self, ui: &mut Ui) -> Response {
+        self.paint_button_indented(ui, paint_default_icon)
+    }
+
+    fn paint_button_indented(
```

Comment:
```suggestion

    /// Will toggle when clicked, etc.
    fn show_button_indented(
```

(it does more than just paint)